eng/Version.Details.xml

```diff
@@ -355,6 +355,36 @@
       <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-aspnetcore</Uri>
       <Sha>4442a188f9200a57635373dcd640893c0e8dcc78</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.DotNet.Test.ProjectTemplates.2.1" Version="1.0.2-beta4.22406.1">
+      <Uri>https://github.com/dotnet/test-templates</Uri>
+      <Sha>0385265f4d0b6413d64aea0223172366a9b9858c</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.DotNet.Test.ProjectTemplates.5.0" Version="1.1.0-rc.23558.1">
+      <Uri>https://github.com/dotnet/test-templates</Uri>
+      <Sha>307b8f538d83a955d8f6dd909eee41a5555f2f4d</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.DotNet.Test.ProjectTemplates.6.0" Version="1.1.0-rc.24069.1">
+      <Uri>https://github.com/dotnet/test-templates</Uri>
+      <Sha>becc4bd157cd6608b51a5ffe414a5d2de6330272</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.DotNet.Test.ProjectTemplates.7.0" Version="1.1.0-rc.24069.1">
+      <Uri>https://github.com/dotnet/test-templates</Uri>
+      <Sha>becc4bd157cd6608b51a5ffe414a5d2de6330272</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.DotNet.Test.ProjectTemplates.8.0" Version="1.1.0-rc.24202.1">
+      <Uri>https://github.com/dotnet/test-templates</Uri>
+      <Sha>49c9ad01f057b3c6352bbec12b117acc2224493c</Sha>
+    </Dependency>
+    <Dependency Name="Microsoft.DotNet.Test.ProjectTemplates.9.0" Version="1.1.0-rtm.24606.1">
+      <Uri>https://github.com/dotnet/test-templates</Uri>
+      <Sha>8d99bca98e3fd0c8f4e10eb200aed20d9665de0e</Sha>
+    </Dependency>
+    <!-- Intermediate is necessary for source build. -->
+    <Dependency Name="Microsoft.SourceBuild.Intermediate.test-templates" Version="1.1.0-rtm.24606.1">
+      <Uri>https://github.com/dotnet/test-templates</Uri>
+      <Sha>8d99bca98e3fd0c8f4e10eb200aed20d9665de0e</Sha>
+      <SourceBuild RepoName="test-templates" ManagedOnly="true" />
+    </Dependency>
```

Comment:
```suggestion
```

Remove test templates : see from @/marcpopMSFT that those should not be included in 9.0.2xx: [Move test templates to SDK repo by Evangelink · Pull Request #44417 · dotnet/sdk](https://github.com/dotnet/sdk/pull/44417/files#diff-fb62e94a1d6f29f863e3d0a22aa38269f6cd1d7f03b109dc06e2cbf2548b86d3)